python/mxnet/metric.py

```diff
@@ -503,21 +582,27 @@ class F1(EvalMetric):
     label_names : list of str, or None
         Name of labels that should be used when updating with update_dict.
         By default include all labels.
+    average : str, default 'macro'
+        Strategy to be used for aggregating across micro-batches.
+            "macro": average the F1 scores for each batch
+            "micro": compute a single F1 score across all batches
```

Comment:
Add period at the end. Currently it renders into:
http://mxnet-doc.s3-accelerate.dualstack.amazonaws.com/api/python/metric/metric.html#mxnet.metric.F1